spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/firestore/GcpFirestoreEmulatorAutoConfigurationTests.java

```diff
@@ -102,4 +134,36 @@ void testDefaultConfig() {
               assertThat(channelProvider.getEndpoint()).isEqualTo("firestore.googleapis.com:443");
             });
   }
+
+  @Test
+  void testFirestoreTemplateParent() {
+    contextRunner
+        .withPropertyValues(
+            "spring.cloud.gcp.firestore.project-id=demo",
+            "spring.cloud.gcp.firestore.emulator.enabled=true",
+            "spring.cloud.gcp.firestore.database-id=testdb")
+        .run(
+            context -> {
+              FirestoreTemplate template = context.getBean(FirestoreTemplate.class);
+              assertThat("projects/demo/databases/testdb/documents").isEqualTo(
```

Comment:
Very nit: The error message created if this fails expects `assertThat(actual).isEqualTo(expected)`.

(This is opposite of junit's `assertTrue(expected, actual)`.)

Reply:
Ack